packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -456,7 +456,7 @@ const VirtualListBaseFactory = (type) => {
 					ev.stopPropagation();
 				} else {
 					const {repeat} = ev;
-					const {dimensionToExtent, isPrimaryDirectionVertical} = this.uiRefCurrent;
+					const {dimensionToExtent, isPrimaryDirectionVertical, props: {isHorizontalScrollbarVisible, isVerticalScrollbarVisible}} = this.uiRefCurrent;
```

Comment:
Is this a remnant from an early implementation? We're pulling these both in as props to this component now so accessing via the ref isn't necessary.